tests/rt.sh

```diff
@@ -563,8 +563,11 @@ echo "Start Regression test" >> ${REGRESSIONTEST_LOG}
 echo                         >> ${REGRESSIONTEST_LOG}
 echo "Testing UFSWM Hash:" `git rev-parse HEAD` >> ${REGRESSIONTEST_LOG}
 echo "Testing With Submodule Hashes:" >> ${REGRESSIONTEST_LOG}
+cd ..
```

Comment:
I know the error well. I think newer git installs do OK without this change needed. I'm OK with this change if you need the backwards compatibility, though.